docs/docs/sample-scripts/spo/make-list-or-library-read-only-for-all-users/index.md

```diff
@@ -0,0 +1,71 @@
+---
+tags:
+  - list
+  - library
+  - security
+  - users
+---
+
+# Make a list or library read-only for all users
+
+Author: [Nico De Cleyre](https://www.nicodecleyre.com), Inspired by [Salaudeen Rajack](https://www.sharepointdiary.com/2020/03/sharepoint-online-make-list-read-only-using-powershell.html)
+
+The following script breaks the role inheritance of the given list if it doesn't have unique permissions already. After that it iterates through all role assignment and removes every role assignment that doesn't have the read permission. After that it gives the read permission to that assignment
+
+=== "PowerShell"
+
+    ```powershell
+     param
+    (
+        [Parameter(Mandatory = $true, HelpMessage = "THe site url where the list is located")][string] $SiteURL,
+        [Parameter(Mandatory = $false, HelpMessage = "The name of the list")][string] $ListName
+    )
+
+    try {
+        #Connect to m365
+        $m365Status = m365 status
+        if ($m365Status -match "Logged Out") {
+        Write-Host "Logging in the User!"
+        m365 login --authType browser
+        }
+
+        $List = m365 spo list get -u $SiteURL --title $ListName --withPermissions --properties "HasUniqueRoleAssignments" | ConvertFrom-Json
+
+        #Break Permissions of the List
+        If ($List.HasUniqueRoleAssignments -eq $False)
+        {
+        m365 spo list roleinheritance break --webUrl $SiteURL --listTitle $ListName --confirm
+        }
```

Comment:
let's be consistent and use either `if` or `If`
also does not seem well formated